kafka-test/src/rdkafka_example.cpp

```diff
@@ -32,6 +32,8 @@
  * (https://github.com/edenhill/librdkafka)
  */
 
+// NOTE: THIS FILE IS DEPRECATED AS OF 5/17/2023 AND IS NOT RECOMMENDED FOR USE.
+
```

Comment:
The same as the above.

Reply:
The files in the `kafka-test` directory are separate from the primary project, not referenced anywhere and have not been tested recently to my knowledge.

Reply:
This deprecation note has been removed.